demonstrator-ogc-services/klips-wmts-slider/src/components/BasicSwipe/index.tsx

```diff
@@ -0,0 +1,166 @@
+        setLabelPosition(width);
+
+        const bottomLeft = getRenderPixel(event, [width, mapSize[1]]);
+        const topLeft = getRenderPixel(event, [width, 0]);
+        const topRight = getRenderPixel(event, [mapSize[0], 0]);
+        const bottomRight = getRenderPixel(event, mapSize);
+
+        const context = event.context as CanvasRenderingContext2D;
+
+        if (!context) {
+            return <></>;
+        };
+
+        context.restore();
+
+        context.save();
+        context.beginPath();
+        context.moveTo(topRight[0], topRight[1]);
+        context.lineTo(bottomRight[0], bottomRight[1]);
+        context.lineTo(bottomLeft[0], bottomLeft[1]);
+        context.lineTo(topLeft[0], topLeft[1]);
+        context.closePath();
+        context.clip();
+    });
+
+    layerLeft.on('prerender', function (event: RenderEvent) {
+        const mapSize = map.getSize(); // [width, height] in CSS pixels
+
+        if (!mapSize) {
+            return <></>;
+        };
+
+        // get render coordinates and dimensions given CSS coordinates
+        const width = Math.round(mapSize[0] * (value / 100));
+
+        const bottomLeft = getRenderPixel(event, [0, mapSize[1]]);
+        const topLeft = getRenderPixel(event, [0, 0]);
+        const topRight = getRenderPixel(event, [width, 0]);
+        const bottomRight = getRenderPixel(event, [width, mapSize[1]]);
+
+        const context = event.context as CanvasRenderingContext2D;
+
+        if (!context) {
+            return <></>;
+        };
+
+        context.restore();
+
+        context.save();
+        context.beginPath();
+        context.moveTo(topRight[0], topRight[1]);
+        context.lineTo(bottomRight[0], bottomRight[1]);
+        context.lineTo(bottomLeft[0], bottomLeft[1]);
+        context.lineTo(topLeft[0], topLeft[1]);
+        context.closePath();
+        context.clip();
+    });
+
+    layerRight.on('postrender', function (event) {
+        var context = event.context as CanvasRenderingContext2D;
+
+        if (!context) {
+            return <></>;
+        };
+
+        context.restore();
+    });
+
+    layerLeft.on('postrender', function (event) {
+        var context = event.context as CanvasRenderingContext2D;
+
+        if (!context) {
+            return <></>;
+        };
+
+        context.restore();
+    });
+
+    const onChange = (newValue: number) => {
+        setValue(newValue);
+        map.render();
+    };
+
+    const toggleRight = () => {
+        setValue(0);
+        map.render();
+    };
+
+    const toggleLeft = () => {
+        setValue(99.4);
+        map.render();
+    };
+
+    const top = 7 + 'vh';
+    const padding = 10 + 'px';
+    const left = labelPosition + 'px';
+    const right = labelPosition - 145 + 'px';
+
+    return (
+        <>
+            <Slider
+                className='custom-slider'
+                tooltip={{ open: false }}
+                onChange={onChange}
+                value={value}></Slider>
```

Comment:
Can the slider be more prominent and accessible? When i used it, it was attached to the very left of my screen and difficult to access. Maybe a thin line would also be great, so it indicates, that the line can be dragged.